client/components/Header.jsx

```diff
@@ -45,16 +47,34 @@ export default class Header extends Component {
     document.querySelector('#navbar-collapse li.dropdown').classList.remove('open');
   }
 
-  onKeyUp = (e) => {
-    if (e && e.key === 'Enter') {
-      this.props.onLogout();
+  onKeyUp = (event) => {
+    if (event && event.key === 'Enter') {
+      event.target.click();
     }
   };
 
+  renderCssSwitcher(languageDictionary) {
+    if (this.props.renderCssToggle) {
+      const toggleTextDefault = languageDictionary.toggleStyleSetDefault || 'Use Default Style';
+      const toggleTextAlt = languageDictionary.toggleStyleSetAlternative || 'Use Alternative Style';
+      const text = (window.localStorage && window.localStorage.getItem('delegated-admin:use-alt-css') === 'true') ? toggleTextDefault : toggleTextAlt;
```

Comment:
Would this be an issue with not re-rendering on change, or are there other areas that are handling it.  I ask because it is checking a value in window.localStorage rather than in a state.